src/library_manager/llext_manager.c

```diff
@@ -76,9 +76,10 @@ static int llext_manager_load_data_from_storage(const struct llext *ext,
 						size_t size, uint32_t flags)
 {
 	unsigned int i;
-	int ret = llext_manager_align_map(vma, size, SYS_MM_MEM_PERM_RW);
+	int ret;
 	const elf_shdr_t *shdr;
 
+	ret = llext_manager_align_map(vma, size, SYS_MM_MEM_PERM_RW);
```

Comment:
this change doesn't seem to be need for the following commits either, let's drop